docs/src/examples.md

```diff
@@ -83,9 +83,13 @@ We can use `MeanStdScaling` for that purpose.
 Note that we are mutating the data frame in-place using `apply!`, and the order of columns specified does not matter.
```

Comment:
```suggestion
Note that we are mutating the data frame in-place using `apply!`.
```
Still true, but no longer relevant for this example because it's one column at a time?